src/ApplicationInsights/custom/New-AzApplicationInsightsContinuousExport.ps1

```diff
@@ -79,9 +79,10 @@ function New-AzApplicationInsightsContinuousExport {
         [Parameter()]
         [Microsoft.Azure.PowerShell.Cmdlets.ApplicationInsights.Category('Body')]
         [System.String[]]
+        [Alias('DocumentType ')]
         # The document types to be exported, as comma separated values.
-        # Allowed values include 'Requests', 'Event', 'Exceptions', 'Metrics', 'PageViews', 'PageViewPerformance', 'Rdd', 'PerformanceCounters', 'Availability', 'Messages'.
-        ${DocumentType },
+        # Allowed values include 'Requests', 'Custom Event', 'Exception', 'Metric', 'Page View', 'Page Load', 'Dependency', 'Performance Counter', 'Availability', 'Trace'.
```

Comment:
Please correct it

Reply:
description for this parameter in Set-AzApplicationInsightsContonuousExport was corrected